apps/frontend/src/network/profile/__tests__/Research.test.tsx

```diff
@@ -44,3 +48,92 @@ it('allows editing your own profile', () => {
   );
   expect(getAllByLabelText(/edit/i)).not.toHaveLength(0);
 });
+
+describe('when editing team membership', () => {
+  const userProfile: ComponentProps<typeof Research>['userProfile'] = {
+    ...createUserResponse(),
+    id: '42',
+  };
+
+  const teams: ComponentProps<typeof Research>['teams'] = [
+    {
```

Comment:
Shouldn't this come from the `userProfile` which we can already generate with 1 team?